src/main/java/core/basesyntax/service/impl/FileReaderImpl.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.Reader;
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class FileReaderImpl implements Reader {
+    @Override
+    public List<String> readFile(String path) {
+        List<String> rows = new ArrayList<>();
+        try (BufferedReader reader = new BufferedReader(new FileReader(path))) {
+            String line = reader.readLine();
```

Comment:
The first line of the file is being read but not added to the list. If the first line is not meant to be skipped (e.g., it's not a header), you should add it to the `rows` list before reading the next line.